src/types/gameConsole.ts

```diff
@@ -12,109 +12,583 @@ export default class GameConsole {
    * OnionOS/GarlicOS ROMs go in the /Roms/{onion} directory:
    *  @see https://github.com/OnionUI/Onion/wiki/Emulators
    *
+   * Batocera ROMs go in the roms/{batocera} directory:
+   *  @see https://wiki.batocera.org/systems
+   *
    * @see https://emulation.gametechwiki.com/index.php/List_of_filetypes
    * @see https://emulation.fandom.com/wiki/List_of_filetypes
    */
   private static readonly CONSOLES: GameConsole[] = [
     // Amstrad
-    new GameConsole(/CPC/i, [], undefined, 'Amstrad', 'CPC'),
+    new GameConsole(/CPC/i, [], undefined, "Amstrad", "CPC", "amstradcpc"),
     // Arduboy
-    new GameConsole(/Arduboy/i, ['.arduboy', '.hex'], 'arduboy', 'Arduboy', undefined),
+    new GameConsole(
```

Comment:
guessing this is prettier auto formatting - maybe worth setting up config/etc to make it easier to contribute PRs appropriately? pretty standard in the industry now so I imagine most folks have it running